doc/source/data/getting-started.rst

```diff
@@ -7,8 +7,8 @@ Getting Started
 In this tutorial you will learn how to:
 
 - Create and save a Ray ``Dataset``.
-- How to transform a ``Dataset`` and pass it into other Ray Tasks.
-- How to create a Ray ``DatasetPipeline`` and run transformations on it.
+- How to transform a ``Dataset``.
```

Comment:
I think that we might still want to call out the passing of a Dataset into a Ray task, which we do below.

Reply:
Combined them.